Stratis.Bitcoin.Tests/Utilities/AsyncDictionaryTest.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Moq;
```

Comment:
Great, its not use yet but with the AsyncDictionary we can remove locks in ConcurrentDictionary